pallets/pallet-bonded-coins/src/benchmarking.rs

```diff
@@ -377,12 +381,19 @@ mod benchmarks {
 
 		let max_currencies = T::MaxCurrenciesPerPool::get();
 		#[extrinsic_call]
-		_(origin as T::RuntimeOrigin, pool_id, fungibles_team, max_currencies);
+		_(
+			origin as T::RuntimeOrigin,
+			pool_id.clone(),
```

Comment:
Do not `clone()` as part of the benchmark itself, but do it before.

Reply:
e1670e61c1a127962c55817f2a536f8cf78fa076